sdks/python/apache_beam/io/gcp/datastore/v1new/types.py

```diff
@@ -267,7 +267,10 @@ def from_client_entity(client_entity):
       if isinstance(value, key.Key):
         value = Key.from_client_key(value)
       if isinstance(value, entity.Entity):
-        value = Entity.from_client_entity(value)
+        if value.key:
+          value = Entity.from_client_entity(value)
+        else:
+          value = {k: v for k, v in value.items()}
```

Comment:
Please add test coverage for this line. 
![image](https://user-images.githubusercontent.com/127695/166571761-0359a59d-2938-4c37-966d-13be4d589bad.png)

Reply:
Done.